src/feature/UserList/AddUsers/AddUsers.tsx

```diff
@@ -0,0 +1,178 @@
+import { Button, Form, Input, InputNumber, Select } from 'antd'
+import React, { useEffect, useState } from 'react'
+import { Modal } from '../../../_atoms/Modal'
+import './addUsers.scss'
+import { IUser } from '../../../store/user/type'
+import apiClient from '../../../helper/api'
+import { useAppDispatch, useAppSelector } from '../../../hooks'
+import { loadUsersAsync } from '../../../store/user/usersThunk'
+import { FormData, initialData } from '../constants'
+
+const { Option } = Select
+
+export const AddUser = () => {
+  const dispatch = useAppDispatch()
+
+  const { isLoading } = useAppSelector((state) => state.user)
+
+  const [isModalOpen, setModalState] = useState(false)
+  const [formValues, setFormValues] = useState<IUser | null>(null)
+  const [isDisabled, setIsDisabled] = useState(false)
+
+  const toggleModal = () => setModalState(!isModalOpen)
+  const onClose = () => setModalState(false)
+
+  const createUser = () => {
+    apiClient()
+      .post('users', formValues)
+      .then(() => {
+        dispatch(loadUsersAsync())
+      })
+  }
+
+  const onFinish = (values: FormData) => {
```

Comment:
Do not setup data to local storage. You need just to send a request and if it succeed clear form state and refetch user list